### PR TITLE
HTML and CSS branch pull request

### PR DESCRIPTION
**Pull request by gama1221**
Hello Code reviewer, this is my first project in Microverse. I am requesting a pull request to review what I did in this branch i.e. html-css-branch. 
. Setup HTML & CSS linter files
. Add HTML and CSS files
. Add readme file
. Add heading element to HTML file
. Format the heading using CSS file

### CODE REVIEW

File: hello.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Html and css</title>
+</head>
+
+<body>
+  <h1> Hello Microverse! </h1>
+</body>
+
+</html>
```
Comment (usorfaitheloho) [inline hello.html:14]: Please kindly implement these changes for your project so as to move to the next one 😄 

-  Kindly add a LICENSE file and update the link to the file in the license section of your README file to enable users know they have authorisation to use your project
- according to the project requirement, you are to add a `test.md` file and add it to your `.gitignore` file. please follow the instructions below

Project requirement

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @gama1221  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Added HTML file ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gama1221** [comment]
Hello @usorfaitheloho, Thanks for your constructive comments and suggestions. I just corrected it accordingly.

Thanks again, @usorfaitheloho

**Review by topeogunleye — COMMENTED** [review]
Hi @gama1221, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Added test.md to `.gitignore` file.
* 

 ## Required Changes ♻️
_Check the comments under the review._
- Hi @gama1221,

Thanks for submitting your pull request. However, please note that it is important to wait for approval before merging your pull request. Kindly revert your merge and wait for approval before merging.

Thanks!


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- Please consider customizing your LICENSE file by replacing the placeholder name with your information.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by topeogunleye** [comment]
Hello, @gama1221! If reverting your merged pull request leads to creating a new one, please kindly include the link to this previous pull request in the new pull request description.

**Pull request merged**